### PR TITLE
Add new setting to disable the classloading cache

### PR DESCRIPTION
**Pull request by uschindler**
Fix issue #75

### CODE REVIEW

**Comment by rjernst** [comment]
I just checked this again with 2.13 and the problem still exists, and this PR does indeed workaround the issue.

**Comment by rjernst** [comment]
I just checked this again with 2.13 and the problem still exists, and this PR does indeed workaround the issue.

File: src/main/resources/de/thetaphi/forbiddenapis/gradle/plugin-init.groovy [hunk 1]
```diff
@@ -23,10 +23,19 @@ if (project.plugins.withType(JavaBasePlugin.class).isEmpty()) {
   throw new PluginInstantiationException('Forbidden-apis only works in projects using the java plugin.');
 }
 
+// chck if running in Gradle Daemon?
+// see: http://stackoverflow.com/questions/23265217/how-to-know-whether-you-are-running-inside-a-gradle-daemon
+boolean isGradleDaemon = System.getProperty('sun.java.command', '').startsWith('org.gradle.launcher.daemon.') ||
```
Comment (rjernst) [inline src/main/resources/de/thetaphi/forbiddenapis/gradle/plugin-init.groovy]: Have you checked this in any other JVMs like openjdk or ibm?

**Comment by rjernst** [comment]
The new changes LGTM, thanks!

File: src/main/resources/de/thetaphi/forbiddenapis/gradle/plugin-init.groovy (hunk 1 above)
Comment (uschindler) [inline src/main/resources/de/thetaphi/forbiddenapis/gradle/plugin-init.groovy]: The second part with stack trace should hit on any JVM. The first check is just a shortcut.

**Pull request merged**